fdbserver/storageserver.actor.cpp

```diff
@@ -1279,6 +1284,10 @@ struct StorageServer {
 		this->storage.kvGets = &counters.kvGets;
 		this->storage.kvScans = &counters.kvScans;
 		this->storage.kvCommits = &counters.kvCommits;
+
+		if (SERVER_KNOBS->BG_METADATA_SOURCE != "tenant") {
+			blobConn = BlobConnectionProvider::newBlobConnectionProvider(SERVER_KNOBS->BG_URL);
+		}
```

Comment:
we should assert false in the opposite case for now, until it is handled properly

Reply:
added assert in tryGetRangeFromBlob. If the knob is enabled but we don't have a connection, the assertion will be triggered.